ui-v2/app/models/proxy.js

```diff
@@ -0,0 +1,12 @@
+import Model from 'ember-data/model';
+import attr from 'ember-data/attr';
+
+export const PRIMARY_KEY = 'uid';
+export const SLUG_KEY = 'ID';
+export default Model.extend({
+  [PRIMARY_KEY]: attr('string'),
+  [SLUG_KEY]: attr('string'),
+  ServiceName: attr('string'),
+  ServiceID: attr('string'),
+  ServiceProxyDestination: attr('string'),
+});
```

Comment:
We introduce a new 'noun' or model here - a Proxy, a 'special' type of Service Instance